optuna/study/study.py

```diff
@@ -659,6 +659,10 @@ def set_user_attr(self, key: str, value: Any) -> None:
 
             See :attr:`~optuna.study.Study.user_attrs` for related attribute.
 
+        .. seealso::
+
+            See :ref:`attributes`.
+
```

Comment:
This is rendered as follows:

![image](https://user-images.githubusercontent.com/3255979/139790188-cdca625a-9a1a-4808-892e-7c3d3a777ca8.png)

It may not be descriptive enough, so how about adding some words?
The following is just an example:
```
See the recipe on :ref:`attributes`.
``` 

@nzw0301 What do you think of it?

Reply:
Indeed, I agree with you.